utils/cTimeManager.cpp

```diff
@@ -242,7 +243,7 @@ void cTimeManager::handleTimerUnits() {
                 }
             }
 
-            map.think_minimap();
+            drawManager->think();
```

Comment:
👍 Looks more consistent this way.

Reply:
yes, for now this is fine. I think `drawManager` should eventually be the `combat state` object that does not exist yet. And the drawing of the `combat` (ie, actual playing of the mission) will be what the `drawManager` is now.